frontend/src/modules/Shares/components/ShareBoxList.js

```diff
@@ -0,0 +1,489 @@
+import {
+  Autocomplete,
+  Box,
+  Checkbox,
+  Container,
+  Grid,
+  TextField,
+  Typography
+} from '@mui/material';
+import CircularProgress from '@mui/material/CircularProgress';
+import CheckBoxOutlineBlankIcon from '@mui/icons-material/CheckBoxOutlineBlank';
+import CheckBoxIcon from '@mui/icons-material/CheckBox';
+import PropTypes from 'prop-types';
+import { useCallback, useEffect, useState } from 'react';
+import { Helmet } from 'react-helmet-async';
+import { Defaults, Pager, ShareStatus, useSettings } from 'design';
+import { SET_ERROR, useDispatch } from 'globalErrors';
+import {
+  listDatasetShareObjects,
+  getShareRequestsToMe,
+  useClient
+} from 'services';
+import { getShareRequestsFromMe } from '../services';
+
+import { ShareBoxListItem } from './ShareBoxListItem';
+import { listDatasets } from '../../Datasets/services';
+import { listAllGroups } from '../../Environments/services';
+//TODO MANAGE DEPENDENCY
```

Comment:
Minor - clean up TODO

Is this in reference to dependency this view needs for Dataset to be loaded for this view to work?

Reply:
Have a  look at my last commit, By the way that the frontend modularization design was built, we will soon run into a non-modularized frontend that has everything int he shared services